src/test/Components/CreateImageWizard/CreateImageWizard.test.tsx

```diff
@@ -636,9 +632,9 @@ describe('Step Registration', () => {
     await clickNext();
     await clickNext();
     await clickNext();
-    await clickNext();
-    await enterBlueprintName();
-    await clickNext();
+    //  await clickNext();
+    //  await enterBlueprintName();
+    // await clickNext();
```

Comment:
Sorry for not being more specific, I've meant we can remove all of the commented lines :sweat_smile: So these...